megamek/src/megamek/server/GameManager.java

```diff
@@ -3680,8 +3680,13 @@ private void loadUnit(Entity loader, Entity unit, int bayNumber) {
         // Load the unit. Do not check for elevation during deployment
         boolean checkElevation = !getGame().getPhase().isLounge()
                 && !getGame().getPhase().isDeployment();
-        loader.load(unit, checkElevation, bayNumber);
-
+        try {
+            loader.load(unit, checkElevation, bayNumber);
+        }catch(IllegalArgumentException e){
```

Comment:
Hate to be that guy, but can we put spaces between the curly braces and the catch statement?

Reply:
done!